ownphotos/settings.py

```diff
@@ -228,11 +240,24 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
+# Allow to define data folder like /var/lib/librephotos
+
+BASE_DATA = ''
+if 'BASE_DATA' in os.environ:
+	BASE_DATA = os.environ['BASE_DATA']
+
+BASE_LOGS='/logs/'
+if 'BASE_LOGS' in os.environ:
+	BASE_LOGS = os.environ['BASE_LOGS']
```

Comment:
you can use a one liner here:

`BASE_LOGS = os.environ.get('BASE_LOGS', '/logs')`

also making `/var/logs/librephotos` the defaults maybe a good idea. this is the standard in many (all ?) linux distribution.